README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
+    mscs-maximum-memory=2048M
+    mscs-server-location=/opt/mscs/server
+    mscs-server-command=$JAVA -Xms$INITIAL_MEMORY -Xmx$MAXIMUM_MEMORY -jar $SERVER_LOCATION/$SERVER_JAR $SERVER_ARGS
+
+Run a Minecraft version 1.6.4 server:
+
+    mscs-client-version=1.6.4
+    mscs-server-version=1.6.4
+
+Use Forge to run a 1.8.4 server (requires additional setup):
+
+    mscs-client-version=1.8.4
+    mscs-server-version=1.8.4
+    mscs-server-jar=forge-1.8-11.14.1.1419-universal.jar
+    mscs-server-url=http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.8-11.14.1.1419/forge-1.8-11.14.1.1419-universal.jar
+
+Use the latest BungeeCord successful build (requires additional setup):
 
+    mscs-server-jar=BungeeCord.jar
+    mscs-server-url=http://ci.md-5.net/job/BungeeCord/lastSuccessfulBuild/artifact/bootstrap/target/BungeeCord.jar
 
-## Usage
+##### Additional documentation
+
+More examples and documentation on server customization can be found on the [wiki](https://github.com/sandain/MinecraftServerControlScript/wiki/Server-Customization-Examples) page.
+
+### The msctl file
+The `msctl` file can be found at `/usr/local/bin`.
+
+The `msctl` file is an additional file for adjusting the properties of your server that are not found in `mscs.properties`. Starting at line 236 and ending at line 385 is where the settings are in this file.
+
+You can set the default settings of **mscs.properties** in this file, as well as some other useful settings:
+* Line 305 is the beginning of the **server instance configuration** settings.
+* Line 321 is the beginning of the **backup configuration** settings.
+* Line 334 is the beginning of the **server log configuration** settings.
+* Line 341 is the beginning of the **listing option** settings.
+* Line 348 is the beginning of the **mirror image** settings.
+* Line 370 is the beginning of the **minecraft overviewer** settings
+
+## Automated backups and restarts
+
+### Scheduling backups
+MSCS uses [**cron**](https://en.wikipedia.org/wiki/Cron), a scheduler software that can run programs on a set interval of time. Below is an example of one way how you could setup backups via cron to backup a world every 2 hours:
+
+Type the following (in any directory): 
+
+  ````
+  export EDITOR=vim
+  crontab -e
+  ````
+  Page down until you get to an empty line. Then paste the following:
+  
+  ````
+  0 */2 * * *  PATH-TO-MSCS backup myWorld
+  ````
+  * `0 */2 * * *` is the time interval to backup. This particular expression means backup every 2 hours. You can change this to 3, 4, 5 or whatever amount of hours to backup X amount of hours. You can also backup according to days, minutes, seconds, the time of the day, and more. See [the wiki page](https://github.com/sandain/MinecraftServerControlScript/wiki/Backup-and-Restore) for more information.
+  * `PATH-TO-MSCS` is the path of where the `mscs` file is (which will be inside the `MinecraftServerControlScript` folder that you downloaded)
```

Comment:
I think the README text should point to `mscs`, not `msctl`, but I do agree that it makes more sense to replace `PATH-TO-MSCS` with the default path to the script, `/usr/local/bin/mscs`.